MKCpu.cpp

```diff
@@ -1717,8 +1831,6 @@ void MKCpu::OpCodeLdyImm()
 	mReg.IndY = mpMem->Peek8bit(GetAddrWithMode(ADDRMODE_IMM));
 	mReg.LastArg = mReg.IndY;
 	SetFlags(mReg.IndY);
-	//SetFlagsRegQ(16);
```

Comment:
Dead code, remove.

Reply:
I just went top-to-bottom and removed every piece of dead code I saw.